cmd/aergocli/cmd/getpeers.go

```diff
@@ -65,12 +78,74 @@ func execGetPeers(cmd *cobra.Command, args []string) {
 		// TODO show long fields
 		res := jsonrpc.ConvLongPeerList(msg)
 		cmd.Println(jsonrpc.MarshalJSON(res))
+	} else if detailed == -2 {
```

Comment:
Is not the command `--detail=2`? So why the negative value here?

Reply:
It just followed previous conventions, though that convention looks strange. this flag has 3 options; 1 or greater show most verbose informations about connected peers, 0 (default) shows slightly less that 1, and -1 shows compact form of informations that is far from detail. I meant -1 as another type of compact form.

Reply:
On the PR description you wrote `--detail=2`

Reply:
It was a typo, I fixed it just now.